components/Video.tsx

```diff
@@ -0,0 +1,16 @@
+import { HTMLAttributes } from 'react'
+
+export type VideoProps = { src: string } & HTMLAttributes<HTMLVideoElement>
+
+export const Video = ({ src, ...props }: VideoProps) => {
+  return (
+    <figure sx={{ paddingBottom: `${100 / (16 / 9)}%` }} {...props}>
+      <iframe
+        src={src}
+        frameBorder="0"
```

Comment:
Can you please check if this attribute changes anything (ideally in different browsers)? I have a feeling it's not necessary.

Reply:
Yes, it is not necessary. Have removed it. :)